src/pool/pool_disjoint.c

```diff
@@ -29,10 +28,30 @@
 
 static char *DEFAULT_NAME = "disjoint";
 
+enum {
+    DP_OVERRIDE_SLAB_MIN_SIZE = 1 << 0,
```

Comment:
if you initialize the bitmask to 0 does it mean that by default slab min size is always overriden?